### ISSUE TITLE
Add detection of router.beforeEach() without calls to next()

### ISSUE THREAD

**Issue by thiderman** [vuejs/vue-migration-helper#38]
I just spent an embarrassingly large amount of time trying to debug why nothing worked during my migration from 1.0.28. Turns out that the added requirement of the `router.beforeEach()` methods having to resolve the hook manually with a call to `next()` was the culprit.

As far as I can see, there is nothing in this project that alerts about this. Now, I lack the vue/js savvy to add that myself, but I suppose this could be of help to others who were using the `beforeEach()` hooks in 1.0.

Cheers for a fantastic tool anyway!

**Comment by chrisvfritz** [comment]
Thanks for reporting this (and sorry about the time you lost 😅 )! What would you think about the warning below?

```
1. Add next as the 3rd argument of router.beforeEach, then call it to continue resolving the route
  Line 10: ./path/to/file.js
  Reason: router.beforeEach can now be asynchronous, so requires the "next" callback in order to resolve
  More info: http://vuejs.org/guide/migration-vue-router.html#router-beforeEach
```

**Comment by thiderman** [comment]
Yes, that warning sounds great! It would definitely have saved me the time, haha!

(sorry for the late-ish reply - vacations happened)
